### PR TITLE
loader: fix nil map after clone

### PR DESCRIPTION
**Pull request by lance6716**
### What problem does this PR solve? 
fix nil map after clone. We may found more map or develop a safe clone in future

### Check List 

Tests 

Code changes

Side effects

Related changes

 - Need to cherry-pick to the release branch

### CODE REVIEW

**Review by csuzhangxc — APPROVED** [review]
LGTM

File: loader/loader.go [hunk 1]
```diff
@@ -465,6 +465,9 @@ func (l *Loader) Init(ctx context.Context) (err error) {
 	if err != nil {
 		return err
 	}
+	// fix nil map after clone
+	// TODO: we may develop `TrueClone` in future
+	lcfg.To.Session = make(map[string]string)
```
Comment (csuzhangxc) [inline loader/loader.go:470]: if `lcfg.To.Session` has some valid values, this `make` will clear them, right?

**Review by csuzhangxc — COMMENTED** [review]
LGTM

**Review by GMHDBJD — APPROVED** [review]
LGTM

**Pull request merged**